src/index.js

```diff
@@ -37,7 +39,7 @@
    * @public
    */
   DayScheduleSelector.prototype.renderHeader = function () {
-    var stringDays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat']
+    var stringDays  = this.options.stringDays
```

Comment:
This is a good idea, and we should figure out a way to do internationalization for other labels, too. :+1: